packages/lib/server-only/user/get-user-by-email.ts

```diff
@@ -9,5 +9,8 @@ export const getUserByEmail = async ({ email }: GetUserByEmailOptions) => {
     where: {
       email: email.toLowerCase(),
     },
+    include: {
+      VerificationToken: true,
```

Comment:
Should we do this or fetch it in a separate method for usage in auth-options?

e.g. `getRecentVerificationToken({ userId }) => Promise<VerificationToken | null>`